packages/@react-aria/list/src/useList.ts

```diff
@@ -52,14 +47,12 @@ export interface ListViewAria {
  * @param ref - The ref attached to the list element.
  */
 export function useList<T>(props: AriaListOptions<T>, state: ListState<T>, ref: RefObject<HTMLElement>): ListViewAria {
-  // Rough copy of useGrid, but modifications + things removed for ListView specific case
   let {
     isVirtualized,
     keyboardDelegate,
-    getRowText = (key) => state.collection.getItem(key)?.textValue,
+    getRowText,
```

Comment:
No default provided here explicitly contrary to what the AriaListOptions state, but the default is setup in the  useGridSelectionAnnouncement hook